tests/test_reduction_ops.py

```diff
@@ -137,6 +138,7 @@ def test_accuracy_cross_entropy_loss_indices(
     gems_assert_close(res_in_grad, ref_in_grad, dtype, reduce_dim=shape[dim])
 
 
+@pytest.mark.skip("random error")
```

Comment:
what's the absolute difference between result and reference?

Reply:
already fixed in our latest triton but in this time test, we still provided elder version of triton, just make it skip in this time test, is it okay for you?